Source: http://pkgs.fedoraproject.org/cgit/kdelibs.git/tree/kdelibs-4.6.90-kstandarddirs.patch
Upstream: Yes, bug #300997
Reason: introduce /etc/kde for settings which overrides defaults in /usr/share
--- kdecore/kernel/kstandarddirs.cpp.me        2011-06-28 09:43:49.000000000 +0200
+++ kdecore/kernel/kstandarddirs.cpp   2011-06-28 09:47:44.000000000 +0200
@@ -1170,6 +1172,11 @@ QStringList KStandardDirs::KStandardDirs
                         if ((local || testdir.exists()) && !candidates.contains(path))
                             candidates.append(path);
                     }
+                    // special-case "config" (forward porting Chris Cheney's
+                    // hack) - we want /etc/kde after the local config paths
+                    // and before the ones in /usr (including kde-profile)
+                    if (local && !strcmp("config", type))
+                       candidates.append(QLatin1String("/etc/kde/"));
                     local = false;
                 }
             else
